backend/build.gradle.kts

```diff
@@ -135,4 +140,19 @@ fun setUpOpenApiGenerator() {
 
 dependencyCheck {
     failBuildOnCVSS = 0.0f
+    suppressionFile = "cve-suppress.xml"
+}
+
+dockerCompose {
+    useComposeFiles.set(
+        listOf("../docker-compose.yml")
+    )
+    isRequiredBy(tasks.test)
+}
+
+flyway {
```

Comment:
We need this to be able to manipulate/repair flyway migrations using gradle without starting an entire spring boot app

Reply:
This is a very nice touch.